frontend/src/views/Play/Player.vue

```diff
@@ -17,6 +19,7 @@ const props = defineProps<{
 const romRef = ref<DetailedRom>(props.rom);
 const saveRef = ref<SaveSchema | null>(props.save);
 const stateRef = ref<StateSchema | null>(props.state);
+const emitter = inject<Emitter<Events>>("emitter");
```

Comment:
is this used?

Reply:
I'll check, probably we have now some variables not being used anymore due to the refactor